client/src/components/Events/EventItem.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Item, Label, Button } from 'semantic-ui-react'
+
+class EventItem extends React.Component {
+  onEdit = () => {
+    const event = {
+      id: this.props.event.id,
+      title: this.props.event.title,
+      description: this.props.event.description,
+      startEpoch: this.props.event.startEpoch,
+      endEpoch: this.props.event.endEpoch,
+      location: this.props.event.location,
+      beginner: this.props.event.beginner,
+      category: this.props.event.category
+    };
+
+    this.props.onEdit(event);
+  }
+
+  render() {
+    const { event} = this.props;
+    if(event.img === "") {
+        // TODO: replace w/ local asset
+        event.img = "http://semantic-ui.com/images/wireframe/image.png";
+    }
+
+    let labels = [];
```

Comment:
You can replace this logic with a call to `Array.prototype.map`. Also, every React element in a list should have a unique `key` prop.

Also thinking that category should be renamed categories.